---
hide:
  - toc
---

<img src="/pics/kevin.jpg" width="250" align="right">

08S06

+49-3641-9-46484

kevin.lamkiewicz[at]uni-jena[dot].de

---

### Main research focus
During my PhD time, my research was focused on RNA secondary structure prediction (and functional annotation) in
RNA viruses. Together with many great collaboration partners I was able to participate in studies that, for example, indicate that the disruption
of a small structural element in the 5' UTR of coronaviruses lead to a complete shutdown of viral replication.  
To facilitate studies in the field of RNA structures in viruses, I develop tools and pipelines to automatically identify such structural elements
in novel viruses.  

Further, I am interested in regulatory elements in viruses in general, regardless of the molecule being RNA or not. In the end, I want to provide a global overview of the interaction network
of viral elements with the host, but also the host response in case of an infection. From this, new hypothesis and maybe even approaches for therapies can be derived.


---

### Publications

#### 2021

- *"Rfam 14: expanded coverage of metagenomic, viral, and microRNA families"*  
  I. Kalvari, E. P. Nawrocki, N. Ontiveros-Palacios, J. Argasinska, **K. Lamkiewicz**, M. Marz, S. Griffiths-Jones, C. Toffano-Nioche,
  D. Gautheret, Z. Weinberg, E. Rivas, S. R. Eddy, R. D. Finn, A. Bateman and A. I. Petrov  
  [Nucleic Acids Research](https://doi.org/10.1093/nar/gkaa1047), Volume 49, Issue D1, D192-D200

- *"Computational strategies to combat COVID-19: useful tools to accelerate SARS-CoV-2 and coronavirus research"*  
  F. Hufsky, **K. Lamkiewicz**, A. Almeida, A. Aouacheria, C. Arighi, A. Bateman, J. Baumbach, N. Beerenwinkel, C. Brandt, M. Cacciabue, S. Chuguransky, O. Drechsel, R. D. Finn, A. Fritz, S. Fuchs, G. Hattab, A. Hauschild, D. Heider, M. Hoffmann, M. Hölzer, S. Hoops, L. Kaderali, I. Kalvari, M. von Kleist, R. Kmiecinski, D. Kühnert, G. Lasso, P. Libin, M. List, H. F. Löchel, M. J. Martin, R. Martin, J. Matschinske, A. C. McHardy, P. Mendes, J. Mistry, V. Navratil, E. P.Nawrocki, Á. O’Toole, N. Ontiveros-Palacios, A. I.Petrov, G. Rangel-Pineros, N. Redaschi, S. Reimering, K. Reinert, A. Reyes, L. Richardson, D. L. Robertson, S. Sadegh, J. B.Singer, K. Theys, C. Upton, M. Welzel, L. Williams, and M. Marz  
  [Briefings in Bioinformatics](https://doi.org/10.1093/bib/bbaa232),  Volume 22, Issue 2, Pages 642–663


#### 2020

-	*"Comparative genome analysis of 33 Chlamydia strains reveals characteristic features of Chlamydia psittaci and closely related species"*  
  M. Hölzer, L.-M. Barf, **K. Lamkiewicz**, F. Vorimore, M. Lataretu, A. Favaroni, C. Schnee, K. Laroucau, M. Marz, and K. Sachse  
  [Pathogens](https://doi.org/10.3390/pathogens9110899), 9(11):899

-	*"HIV-1 lethality and loss of Env protein expression induced by single synonymous substitutions in the virus genome intronic splicing silencer"*  
  A. Jordan-Paiz, M. Nevot, **K. Lamkiewicz**, M. Lataretu, S. Franco, M. Marz, and M. A. Martinez:  
  [Journal of Virology](http://doi.org/10.1128/JVI.01108-20), 94 (21) e01108-20

-	*"A comprehensive annotation and differential expression analysis of short and long non-coding RNAs in 16 bat genomes"*  
  N. Mostajo, M. Lataretu, S. Krautwurst, F. Mock, D. Desiro, **K. Lamkiewicz**, M. Collatz, A. Schoen, F. Weber, M. Marz and M. Hölzer  
  [NAR Genomics and Bioinformatics](http://doi.org/10.1093/nargab/lqz006),  vol. 2, iss. 1, p. lqz006

- *"Virus Bioinformatics"*  
  N. Pappas, S. Roux, M. Hölzer, **K. Lamkiewicz**, F. Mock, M. Marz, B. E. Dutilh  
  [Reference Module in Life Sciences](https://doi.org/10.1016/B978-0-12-814515-9.00034-5}), Elsevier

-	*"The International Virus Bioinformatics Meeting 2020."*  
  F. Hufsky, N. Beerenwinkel, I. M. Meyer, S. Roux, G. M. Cook, C. M. Kinsella, **K. Lamkiewicz**, M. Marquet, D. F. Nieuwenhuijse,
	I. Olendraite, S. Paraskevopoulou, F. Young, R. Dijkman, B. Ibrahim, J. Kelly, P. Le Mercier, M. Marz, A. Ramette, and V. Thiel  
  [Viruses](https://doi.org/10.3390/v12121398), 12, 1398

#### 2019

- *"Direct RNA nanopore sequencing of full-length coronavirus genomes provides novel insights into structural variants and enables modification analysis"*  
  A. Viehweger, S. Krautwurst, **K. Lamkiewicz**, R. Madhugiri, J. Ziebuhr, M. Hölzer and M. Marz 	
  [Genome Research](https://doi.org/10.1101/gr.247064.118), p. 483693

-	*"A CRISPR activation screen identifies genes protecting from Zika virus infection."*  
  A. Dukhovny, **K. Lamkiewicz**, Q. Chen, M. Fricke, N. Jabrane-Ferrat, M. Marz, J. U. Jung, and E. H. Sklan  
  [Journal of Virology](http://doi.org/10.1128/JVI.00211-19),  93 (16) e00211-19

-	*"Structure and hierarchy of influenza virus models revealed by reaction network analysis."*  
  S. Peter, M. Hölzer, **K. Lamkiewicz**, P. S. di Fenizio, H. A. Hwaeer, M. Marz, S. Schuster, P. Dittrich, and B. Ibrahim  
  [Viruses](http://doi.org/10.3390/v11050449), vol. 11, iss. 5, p. 449

- *"The third annual meeting of the European Virus Bioinformatics Center."*  
  F. Hufsky, B. Ibrahim, S. Modha, M. R. J. Clokie, S. Deinhardt-Emmer, B. E. Dutilh, S. Lycett, P. Simmonds,
	V. Thiel, A. Abroi, E. M. Adriaenssens, M. Escalera-Zamudio, J. N. Kelly, **K. Lamkiewicz**, L. Lu, J. Susat,
	T. Sicheritz, D. L. Robertson, and M. Marz	
  [Viruses](http://doi.org/10.3390/v11050420), vol. 11, iss. 5, p. 420

#### 2018

-	*"Structural and functional conservation of cis-acting RNA elements in coronavirus 5'-terminal genome regions."*  
  R. Madhugiri, N. Karl, D. Petersen, **K. Lamkiewicz**, M. Fricke, U. Wend, R. Scheuer, M. Marz, and J. Ziebuhr  
  [Virology](http://doi.org/10.1016/j.virol.2017.11.025), vol. 517, p. 44–55

- *"Identification of potential microRNAs associated with Herpesvirus family based on bioinformatic analysis.”*  
  **K. Lamkiewicz**, E. Barth, M. Marz, and B. Ibrahim  
	[bioRxiv](http://doi.org/10.1101/417782), p. 417782